chaoslib/powerfulseal/pod_failure_by_powerfulseal.yml

```diff
@@ -0,0 +1,44 @@
+- name: Generate the powerfulseal deployment spec from template
+  template:
+    src: /chaoslib/powerfulseal/powerfulseal.j2
+    dest: powerfulseal.yml
+
+- name: Setup powerfulful to initiate random pod chaos 
+  shell:
+    kubectl apply -f powerfulseal.yml -n {{ a_ns }}
+  args:
+    executable: /bin/bash
+  register: result
+  failed_when: "result.rc != 0"
+
+- name: Confirm that powerfulseal pod is running
+  shell: >
+    kubectl get pod -l name=powerfulseal --no-headers -n {{ a_ns }}
```

Comment:
Is there any chance like one namespace have multiple powerfulseal pod? if yes then what was the impact

Reply:
Good question!! It is undesirable at this point @chandankumar4 , but technically it is possible if multiple chaosengines are created for different apps in same ns and reuse the same experiment CR. This will need us to create instance specific names for powerfulseal deploy as well as the configmap

Reply:
okay!

Reply:
This can be achieved by pushing a unique "RUN_ID" ENV (with value: engine-name+hash) for every new job launched by the executor (operator). All resources created as part of this chaosengine -> job/test-names, the libs used by the jobs (such as pumba, powerfulseals) etc.., need to carry this RUN_ID stamp.

Reply:
Sounds good!!